ide/gsf.testrunner.ui/nbproject/project.xml

```diff
@@ -136,7 +145,7 @@
                     <build-prerequisite/>
                     <compile-dependency/>
                     <run-dependency>
-                        <specification-version>9.0</specification-version>
+                        <implementation-version/>
```

Comment:
Turning normal dependency into implementation one. That's unfortunate. What's the reason?

Reply:
`org.netbeans.modules.parsing.impl.indexing.CacheFolder.getDataFolder(...)` call is necessary to retrieve data from indexer caches. Official API in `parsing.indexing` has not created yet, as the list of its clients is small and it should not be publicly available IMHO.